README.mkdn

```diff
@@ -14,12 +23,14 @@ is an engine - not a standalone app.
 
 [Discourse]: http://www.discourse.org/
 [Forem]: https://www.github.com/radar/forem
-[Tectura]: https://github.com/caelum/tectura
-[Heterotic Beast]: https://github.com/distler/heterotic_beast
-[Altered Beast]: https://www.github.com/courtenay/altered_beast
 
+Thredded works with SQLite, MySQL (v5.6.4+), and PostgreSQL.
+Thredded has no infrastructure dependencies other than the database and (optionally) an ActiveJob backend.
```

Comment:
If there is no ActiveJob backend (such as Redis) set, ActiveJob will just run the jobs inline.